pulpcore/tasking/tasks.py

```diff
@@ -126,8 +128,20 @@ def _execute_task(task):
         _logger.info("\n".join(traceback.format_list(traceback.extract_tb(tb))))
         send_task_notification(task)
     else:
+        finish = time.perf_counter_ns()
+        # perf_counter_ns returns the time in nanoseconds,
+        # we divide by 1e+6 to obtain it in milliseconds
+        execution_time = (finish - start) / 1000000
```

Comment:
This is an optional idea. Consider putting the nanosecond aspect as a comment after this instead of two lines before. Comments can be tricky because sometimes code moves around in the comments don't follow them 


```suggestion
        execution_time = (finish - start) / 1000000 # convert ns to s
```

Reply:
"ns to ms"